interfaces/HO-Portal/src/app/components/update-property-item/update-property-item.component.html

```diff
@@ -34,6 +34,44 @@
           pattern="[0-9]*"
         ></ion-input>
       </ng-container>
+      <ng-container *ngSwitchCase="'fsp'">
```

Comment:
I would suggest actually to not include this whole FSP-specific block in this generic "update-property-item"-component.

But instead create a new component for it, or directly in: `edit-person-affected-popup.component.html`
https://github.com/global-121/121-platform/pull/2391/files#diff-7f2096eb325da4786303a7ecf9a059f8e3a5b49f65fcdca62aed7f32251356d4R61-R67